en/docs/learn/api-controller/ci-cd-with-wso2-api-management.md

```diff
@@ -247,6 +247,44 @@ The **apictl** tool should be installed in the automation servers to begin the p
     -   For example, if an API is in the `PUBLISHED` state in the development environment, it will also be in the same state 
     in the testing environment. This default behavior can be changed via the **apictl** tool, which assigns APIs the `CREATED` state after importing. 
 
+<a name="G"></a>
+### (G.) - Get keys for an API
+
+Follow the instructions below to generate a JWT/OAuth token for testing purposes using CTL in order to invoke an API by subscribing to it using the default application:
```

Comment:
I think this is not the DefaultApplication but a new app created by CTL. Shall we do a small tweak to correct this?